Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.8.6-alpine
+
+## Prepare docker base dependency
+RUN set -euxo pipefail; \
```

Comment:
I _think_ this is not necessary in this context, as `docker build` would fail anyway (at least when using `&&` instead of `;` to separate commands), but I also don't know why it shouldn't be here.

Reply:
I understood this as best practice, because almost every major Dockerfile does it like this

Reply:
Good enough for me :D Like I said I don't see a reason why not; just looks odd to me :D